examples/templates/exposed-thing/src/base.ts

```diff
@@ -104,21 +108,35 @@ export class WotDevice {
         });
     }
 
-    private myPropertyHandler() {
+    private myPropertyReadHandler() {
         return new Promise((resolve, reject) => {
             // read something
-            resolve();
+            resolve(this.myProperty);
         });
     }
 
-    private myActionHandler(inputData) {
+    private myPropertyWriteHandler(inputData, options?) {
         return new Promise((resolve, reject) => {
-            // do something with inputData
-            resolve();
+            // write something to property
+            this.myProperty = inputData;
+            // resolve that write was succesful
```

Comment:
it would be good to show emitting observed properties

Reply:
I think we are missing: https://w3c.github.io/wot-scripting-api/#the-emitpropertychange-method . Is going to be added after I've done reviewing: https://github.com/relu91/thingweb.node-wot/pull/17